src/main/java/de/mediathekview/mserver/crawler/phoenix/tasks/PhoenixOverviewTask.java

```diff
@@ -0,0 +1,79 @@
+package de.mediathekview.mserver.crawler.phoenix.tasks;
+
+import com.google.common.reflect.TypeToken;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.basic.SendungOverviewDto;
+import de.mediathekview.mserver.crawler.phoenix.parser.PhoenixSendungOverviewDeserializer;
+import de.mediathekview.mserver.crawler.zdf.tasks.ZdfTaskBase;
+import java.lang.reflect.Type;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.WebTarget;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class PhoenixOverviewTask extends ZdfTaskBase<CrawlerUrlDTO, CrawlerUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(PhoenixOverviewTask.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)